phpcs.xml.dist

```diff
@@ -179,4 +179,11 @@
 		<exclude-pattern>/inc/wpseo-functions-deprecated\.php$</exclude-pattern>
 	</rule>
 
+	<!-- AUTO-GENERATED CODE -->
+
+	<!-- Allow single-line associative arrays in generated files. -->
+	<rule ref="WordPress.Arrays.ArrayDeclarationSpacing.AssociativeArrayFound">
+		<exclude-pattern>/src/config/migrations/20[0-9]{12}_*\.php$</exclude-pattern>
```

Comment:
I'm not sure this exclusion is correct, because I thought only the filenames of the migration files are generated, not the content of them. @herregroen can you please shed a light on this?

Reply:
The content is partly generated but the template of it is fully under our control: https://github.com/Yoast/wordpress-seo/blob/trunk/config/composer/actions.php#L272

From that point on the template is filled in by developers.

In either case I agree that the exclusion isn't correct.

Reply:
IIRC I added that when the migrations folder was first introduced, thinking that all files would have lots of the multi-item associative arrays like the initial file `WpYoastIndexable` and trying reduce code-churn as all those multi-item associative arrays should be multi-line according to YoastCS, which also means that all those function calls need to be proper multi-line.

I'll send in a PR to fix up the code (mostly that first file) and will remove the exclusion.